website/content/en/wgs/infra-lifecycle/glossary/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Glossary
```

Comment:
I wonder if a goal would be to combine and have a top level glossary?

Reply:
That's something I'm happy to address in the future together with the other WG leads.